tests/conftest.py

```
@@ -48,3 +48,18 @@ def create_safe_test_file(content, prefix="test", extension="py", directory=None
             f.write(content)
 
         return filepath
+
+
+def pytest_collection_modifyitems(config, items):
```

Ruff (0.14.8)

Unused function argument: `config`

(ARG001)